modules/hcp-updating-requirements.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+//
+// * hosted-control-planes/hcp-updating.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="hosted-control-planes-upgrading-requirements_{context}"]
+= Requirements to update hosted control planes
+
+The multicluster engine for Kubernetes operator (MCE) can manage one or more {product-title} clusters. After you create a hosted cluster on {product-title}, you must import your hosted cluster in the MCE as a managed cluster. Then, you can use the {product-title} cluster as a management cluster.
+
+Consider the following requirements before you start the update process:
```

Comment:
Global comment regarding **update versus upgrade**.

Software updates modify an existing program. Upgrades replace a program with its next or latest version.
So, I think the assembly talks about upgrading to a new OCP release image. we need to use the correct wording for clarity. Suggestion for the entire assembly.